lib/fizzy/parser.cpp

```diff
@@ -258,9 +262,9 @@ inline parser_result<Code> parse(const uint8_t* pos, const uint8_t* end)
         if (local_count > std::numeric_limits<uint32_t>::max())
             throw parser_error{"too many local variables"};
     }
-    std::get<0>(result).local_count = static_cast<uint32_t>(local_count);
+    result.local_count = static_cast<uint32_t>(local_count);
 
-    return result;
+    return {result, pos3};
```

Comment:
Should use `std::move(result)` here?

Reply:
Yes, I think it will be copied otherwise

Reply:
Done.